agency-of-learning/PairApp
# add initial tailwind config and integrate daisy as component lib

Addresses the first part of #34. Subsequent diffs can/will include a refactoring of the existing styling to use DaisyUI components.

## Change in the diff
The diff shows that a new gem `"tailwindcss-rails"` was added to the Gemfile and a new CSS file "application.tailwind.css" was created in the assets/stylesheets directory. 

Additionally, changes were made to the Procfile.dev, config/tailwind.config.js, app/views/layouts/application.html.erb, and app/assets/config/manifest.js files to support the new CSS file. 

These changes involve adding a new process to the development environment to watch for changes to the CSS file, configuring Tailwind CSS, linking the new CSS file in the application layout file, and adding the new CSS file to the asset pipeline manifest.

## How to test:
Checkout branch, `bundle install` and `yarn install`, then run `bin/dev`. It should work.

Open `landing#index.html`. Add a code snippet that utilizes native daisyUI classes, e.g.
```
<div class="btn-primary">
  hello world, I'm a daisy button
</div>
```

When you refresh the page, a purple button should show up.
